GameData/RealismOverhaul/RO_SuggestedMods/MMSEV/RO_MMSEV.cfg

```diff
@@ -0,0 +1,919 @@
+//	Benjee10_MMSEV_baseNode1-25				PT-M075NA Quad-Node (Horizontal)			2,1x1,9
+//	Benjee10_MMSEV_baseNode_vertical		PT-M075NB Configurable Node Module (Vertical)	2,1x1,9
+//	Benjee10_MMSEV_baseHDU					PT-3A Command Node Module					1,9x3,75	
+//	Benjee10_base_HDU_attic					PT-3H Expandable Habitat Module				1,9x3,75
+//	Benjee10_base_HDU_base					PT-3S Octagonal Structural Support Unit		Structure
+//	Benjee10_MMSEV_baseHDU_hab				PT-3B Habitation Module						1,9x3,75
+//	Benjee10_MMSEV_baseLifter				PT-S20 Hydraulic Jack Unit
+//	Benjee10_MMSEV_baseMount1-875			PT-SA100 Module Adapter (1.875m)			Structure
+//	Benjee10_base_platform					PT-SP100 Structural Support Unit
+//	Benjee10_MMSEV_baseRadiator				PT-Z100R Flat Aspect Radiator Panel
+//	Benjee10_MMSEV_baseRailings				K-100 Kerbal Stabiliser Rails
+//	Benjee10_MMSEV_baseSolarArray			PT-Z100S Flat Aspect Solar Array
+//	Benjee10_base_stairs					K-500 Deployable Staircase
+// 	Benjee10_MMSEV_base_TNode1-25			PT-M075NT T-node Module
+//	Benjee10_MMSEV_baseTurbineBoom			TM-10B Modular Structural Boom (0.3125m)
+//	Benjee10_MMSEV_baseTurbineBoomMount		TM-05B Modular Structural Boom Mount
+//	Benjee10_MMSEV_baseTurbineGiant			TRB-100 Deployable Static Wind Turbine
+//	Benjee10_MMSEV_baseTurbineHelicalLarge1	TRB-0H3-A Helical Wind Turbine
+//	Benjee10_MMSEV_baseTurbineHelicalLarge2	TRB-0H3-B Vertical-Axis Wind Turbine
+//	Benjee10_MMSEV_baseTurbineHelicalSmall	TRB-0H2 Helical Wind Turbine
+//	Benjee10_MMSEV_baseTurbineHelicalTiny	TRB-0H1 Vertical-Axis Wind Turbine
+//	Benjee10_MMSEV_baseTurbineHighDensity	TRB-1D-3 Heavy-Duty Wind Turbine
+//	Benjee10_MMSEV_baseTurbineLowDensity	TRB-1D-1 Low-Density Wind Turbine
+//	Benjee10_MMSEV_baseTurbineMedDensity	TRB-1D-2 Medium-Density Wind Turbine
+//	Benjee10_MMSEV_baseTurbineMotor			TM-10M Yaw Axis Drive (0.3125m)
+//	Benjee10_MMSEV_chassis					Planetside Exploration Vehicle Chassis
+//	Benjee10_MMSEV_grapple					Planetside Exploration Vehicle Grapple Unit
+//	Benjee10_MMSEV_headlight				MU-45 Adjustable Micro Spot Light
+//	Benjee10_MMSEV							Planetside Exploration Vehicle Command Module	3,1x2x1,3
+//	Benjee10_MMSEV_monopropSled				Planetside Exploration Vehicle Service Module
+//	Benjee10_MMSEV_wheelLinear				P-G40L Linear Rover Wheel
+//	Benjee10_MMSEV_wheelMount				P-G40M Rover Wheel Actuating Mount
+//
+//	=================================================================
+@PART[Benjee10_MMSEV_adapter_0-9375]:FOR[RealismOverhaul]:NEEDS[Benjee10_MMSEV]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.8
+
+	@title = 2.25m to 1.6875m Adapter PT-091A
+	@description =  This low-profile adapter provides a connection between 2.25m and 1.6875m bulkhead profiles.
+	@tags = planetside, exploration, technologies, station, radial, attach
+
+	@maxTemp = 773.15
+	@skinMaxTemp = 773.15
+	@crashTolerance = 10
+	
+	@mass = 0.5
+}
+@PART[Benjee10_MMSEV_crewTube0]:FOR[RealismOverhaul]:NEEDS[Benjee10_MMSEV,RealFuels]
```

Comment:
`NEEDS[Benjee10_MMSEV]` is unnecessary since the part won't exist without it.  Remove throughout.

Either remove the RealFuels NEEDS (because RO deps RealFuels anyway) -or- just move it to the only PartModule that requires it: ModuleFuelTanks